lib/taskana-core/src/main/java/pro/taskana/task/api/TaskService.java

```diff
@@ -160,39 +160,86 @@ Task createTask(Task taskToCreate)
-   * @param destinationWorkbasketId The Id of the target work basket
-   * @return the transferred task
+   * @param destinationWorkbasketId The Id of the target Workbasket
+   * @return the transferred Task
    * @throws TaskNotFoundException Thrown if the {@link Task} with taskId was not found.
-   * @throws WorkbasketNotFoundException Thrown if the target work basket was not found.
+   * @throws WorkbasketNotFoundException Thrown if the target Workbasket was not found.
    * @throws NotAuthorizedException Thrown if the current user is not authorized to transfer this
-   *     {@link Task} to the target work basket
-   * @throws InvalidStateException Thrown if the task is in a state which does not allow
+   *     {@link Task} to the target Workbasket
+   * @throws InvalidStateException Thrown if the Task is in a state which does not allow
    *     transferring
    */
-  Task transfer(String taskId, String destinationWorkbasketId)
+  default Task transfer(String taskId, String destinationWorkbasketId)
+      throws TaskNotFoundException, WorkbasketNotFoundException, NotAuthorizedException,
+          InvalidStateException {
+    return transfer(taskId, destinationWorkbasketId, true);
+  }
+
+  /**
+   * Transfer a Task to another Workbasket. <br>
+   * The transfer resets the read flag and sets the transfer flag if {@code setTransferFlag} is
+   * {@code true}.
+   *
+   * @param taskId the id of the {@link Task} to be transferred
+   * @param destinationWorkbasketId the Id of the target Workbasket
+   * @param setTransferFlag the control about whether to set the {@linkplain Task#isTransferred()}
+   *     flag or not
+   * @return the transferred Task
+   * @throws TaskNotFoundException Thrown if the {@link Task} with taskId was not found.
+   * @throws WorkbasketNotFoundException Thrown if the target Workbasket was not found.
+   * @throws NotAuthorizedException Thrown if the current user is not authorized to transfer this
+   *     {@link Task} to the target Workbasket
+   * @throws InvalidStateException Thrown if the Task is in a state which does not allow
+   *     transferring
+   */
+  Task transfer(String taskId, String destinationWorkbasketId, boolean setTransferFlag)
       throws TaskNotFoundException, WorkbasketNotFoundException, NotAuthorizedException,
           InvalidStateException;
 
   /**
-   * Transfer a task to another work basket. The transfer sets the transferred flag and resets the
-   * read flag.
+   * Transfer a Task to another Workbasket. <br>
+   * The transfer sets the transferred flag and resets the read flag.
    *
-   * @param taskId The id of the {@link Task} to be transferred
-   * @param workbasketKey The key of the target work basket
-   * @param domain The domain of the target work basket
-   * @return the transferred task
+   * @param taskId the id of the {@link Task} to be transferred
+   * @param workbasketKey the key of the target Workbasket
+   * @param domain the domain of the target Workbasket
```

Comment:
We should discuss this. Everywhere else we start the Javadoc with a capital letter.

Reply:
Yes, we will discuss it in our COP.